websites/B/Brain.fm/presence.ts

```diff
@@ -0,0 +1,108 @@
+const presence = new Presence({
+		clientId: "981442534167441428"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, hostname } = document.location;
+
+	if (["www.brain.fm", "try.brain.fm"].includes(hostname)) {
+		if (["/", "/home"].includes(pathname))
+			presenceData.details = "Viewing the home page";
+		else if (pathname.startsWith("/about"))
+			presenceData.details = "Viewing the about page";
+		else if (pathname.startsWith("/pdfs"))
+			presenceData.details = "Reading a PDF";
+		else if (pathname.startsWith("/science"))
+			presenceData.details = "Viewing the science page";
```

Comment:
this is vague. What's the science page?

Reply:
What should it be called instead?
https://www.brain.fm/science